### ISSUE TITLE
Assign Targets using labels does not seem to work

### ISSUE THREAD

**Issue by patrickschneider** [hetznercloud/cli#760]
### TL;DR

I created 8 nodes w/o public IP-addresses in 2 different sites and assigned them labels (node-site=fsn1 / node-site=hel1)
There is a Network definition "network-1".
I want to create a Load Balancer using the following hcloud CLI commands:
hcloud load-balancer create --algorithm-type least_connections --location fsn1 --name fsn1-lb1 --type lb11
hcloud load-balancer add-service --protocol tcp --listen-port 1234 --destination-port 1234  fsn1-lb1
hcloud load-balancer add-service --protocol tcp --listen-port 2345 --destination-port 2345  fsn1-lb1
hcloud load-balancer attach-to-network --network network-1 fsn1-lb1
hcloud load-balancer add-target --label-selector node-site=fsn1  fsn1-lb1
If I check the Load Balancer in the Web-UI, the label is showing up, but there are no targets. If I try to add the label "node-site=fsn1" in the Web-UI , it complains, that the label selection is already in use.
If I delete the Target selector and re-add the same label selection it in the Web-UI, the targets show up.

### Expected behavior

The label selector should work and the targets should appear

### Observed behavior

Targets in the Web-UI are empty, same happens when running hcloud load-balancer describe fsn1-lb1

### Minimal working example

hcloud network create --ip-range "10.0.2.0/24" --name network-1
hcloud server create --name fsn1-node1 --image rocky-9 --type cx31 --datacenter fsn1-dc14 --network "network-1" --label 'node-site=fsn1' --without-ipv4 --without-ipv6 --ssh-key "mysshkey"
... add a few other servers
hcloud load-balancer create --algorithm-type least_connections --location fsn1 --name fsn1-lb1 --type lb11
hcloud load-balancer add-service --protocol tcp --listen-port 1234 --destination-port 1234  fsn1-lb1
hcloud load-balancer add-service --protocol tcp --listen-port 2345 --destination-port 2345  fsn1-lb1
hcloud load-balancer attach-to-network --network network-1 fsn1-lb1
hcloud load-balancer add-target --label-selector node-site=fsn1  fsn1-lb1

### Additional information

hcloud 1.43.1

**Comment by patrickschneider** [comment]
Hello,

sorry for the delay... I just checked and the targets are showing up now! 
Thank you!

Best regards,
Patrick
